carveracontroller/main.py

```diff
@@ -331,6 +334,27 @@ def selected_anchor(self):
         elif self.cbx_current_position.active:
             return 4
         return 1
+    
+    def update_offsets(self):
+        # Use the same logic as CoordPopup.load_origin_label to set offsets
+        app = App.get_running_app()
```

Comment:
As you pointed out you can unselect a radio button (I couldn't find a kivy attribution to prevent that).

If you unselect the radio button option it still triggers `update_offsets()` but since there is no active selection the value of `x` and `y` are unset and causes an app crash.

Lets just set an initial value for x and y to prevent this scenario.

```suggestion
        app = App.get_running_app()
        x = 0
        y = 0
```